File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s, string skip, int index) {
    string answer = "";
    string E="abcdefghijklmnopqrstuvwxyz";
    
    for(int i=0; i<skip.size(); i++){
        E.erase(find(E.begin(),E.end(),skip[i]));
    }
    
    for(int i=0; i<s.size(); i++){
        int tmp=E.find(s[i])+index;
        if(tmp >=E.size()) tmp %= E.size();
        answer+=E[tmp];
    }
    return answer;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    aph=['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
    for i in skip :
        aph.remove(i)
        
    for i in s:
        a="".join(aph)
        answer+=aph[(a.find(i)+index)%(26-len(skip))]
    return answer